verification.py

```diff
@@ -377,205 +392,158 @@ def verify_recombination(
         pyplot.close("all")
 
 
-class MsTest(SimulationVerifier):
-    def __init__(self, output_dir):
-        super().__init__(output_dir)
+class MsCmd1(Test):
+    def __init__(self, name, group, command):
+        super().__init__(name, group)
+        self.command = command
+
+    def run(self, output_dir):
+        MsTest().add_ms_instance(self.name, self.command)
```

Comment:
I don't understand how this would work - we'd need to port the code from the verifier infrastructure into here so that the test can actually run. We want to avoid using the old infrastructure for adding test instances, as this won't work in parallel (that's one of the main motivations for this refactor).

Reply:
I mean to do the same thing. But I thought of doing it in steps after I saw that the changes were getting too big for one commit. I mentioned it here too
> I have started with a skeleton for now. It was getting too big for one change so I thought to do it in steps first.

Reply:
That's a fair point, but we do need to keep it working though, and I don't see how what you have currently would. I don't think the diff will be that big, if the classes that you have are mostly kept as is, and just the registration etc changes a bit?

Reply:
Yeah. But I was not sure if I was proceeding the correct direction, so I wanted to get your views on it first. This commit was mainly for tracking and getting my progress evaluated. I'll finally merge all the commits into one that would be a working model.
I'll hopefully get everything correct in the next commit itself.

Reply:
Great, thanks.